de-lib/src/main/java/org/iplantc/de/client/services/GroupServiceFacade.java

```diff
@@ -142,4 +142,11 @@
      * @param callback
      */
     void searchTeams(String searchTerm, AsyncCallback<List<Group>> callback);
+
+    /**
+     * Leave the selected Team
+     * @param group
+     * @param callback
+     */
+    void leaveTeam(Group group, AsyncCallback<List<UpdateMemberResult>> callback);
```

Comment:
This is minor, but please consider more descriptive parameter names.  Maybe `team` instead `group` and `updateMember` instead of `callback`. It helps with the documentation.  Also, the `callback`/`updateMember` parameter should be documented to describe its intent.